src/main/java/hudson/plugins/parameterizedtrigger/FileBuildTriggerConfig.java

```diff
@@ -19,6 +21,7 @@ public FileBuildTriggerConfig(String projects,
     private ResultCondition condition;
     private boolean triggerWithNoParameters;
     private boolean includeCurrentParameters;
+	@SuppressFBWarnings(value="UUF_UNUSED_FIELD", justification="Do not risk compatibility")
     private boolean batchCondition;
```

Comment:
If it is in fact unused that it should be safe to delete?

Reply:
Okay :+1: 
I tried to look for all the config.xml files within this repo and found one occurrence of this class in one of the config.xml [right here](https://github.com/jenkinsci/parameterized-trigger-plugin/blob/master/src/test/resources/hudson/plugins/parameterizedtrigger/test/CompatibilityTest/jobs/old/config.xml#L22). 

But based on your second sentence, then would it be okay to delete this field?

Reply:
Oh perfect, so if you leave https://github.com/jenkinsci/parameterized-trigger-plugin/blob/eb2d40720a4b1e37ea38c1e01fdf91d485fc7e5e/src/test/resources/hudson/plugins/parameterizedtrigger/test/CompatibilityTest/jobs/old/config.xml#L24 alone and the corresponding `@LocalData` test still passes with the field deleted, that is some confirmation that it was safe.